release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1165,15 +1165,16 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 [id="ocp-4-10-RAN-disabling-chronyd"]
 ====  Disable chronyd in the PolicyGenTemplate CR
 
-You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:
+You must disable `chronyd` if you update to {product-title} 4.10 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the `ztp-site-generate` container image:
```

Comment:
This is referring the PolicyGenTemplate CR in the group policies (not the source-crs)
 - fileName: TunedPerformancePatch.yaml
      policyName: "config-policy"
      spec:
        profile:
          - name: performance-patch
            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
            data: |
              [main]
              summary=Configuration changes profile inherited from performance created tuned
              include=openshift-node-performance-openshift-node-performance-profile
              [bootloader]
              cmdline_crash=nohz_full=2-23,26-47
              [sysctl]
              kernel.timer_migration=1
              [service]
              service.stalld=start,enable
              service.chronyd=stop,disable

Reply:
@tliu2021 Thank you for the clarification. 
Sorry that it was not immediately clear to me where the [line](https://github.com/openshift/openshift-docs/pull/49033/files#diff-a5bd860487970afb9da12221d481f6220b28719d13763193710cd76ae315d2a2R1177) was required to be added. If I understand it correctly, you are referring to the group PolicyGenTemplates (or the `group` YAMLs) that can be found at argocd/example/policygentemplates in the `ztp` container image.

Reply:
Yes, the example can be found at argocd/example/policygentemplates. The customers would need to update their own PolicyGenTemplate CR in their git repository.